Lib/asyncio/base_events.py

```diff
@@ -812,7 +812,7 @@ def call_at(self, when, callback, *args, context=None):
         timer = events.TimerHandle(when, callback, args, self, context)
         if timer._source_traceback:
             del timer._source_traceback[-1]
-        heapq.heappush(self._scheduled, timer)
+        heapq.heappush(self._scheduled, (when, timer))
```

Comment:
It seems a bit counterintuitive that this would be faster with the overhead of creating the tuple, but its still far less than all the `__lt__` calls.

Reply:
I feel like that should be a comment in the code.